src/pybind/mgr/dashboard/frontend/src/app/shared/api/user.service.ts

```diff
@@ -19,7 +23,8 @@ export class UserService {
   }
 
   get(username: string) {
```

Comment:
Skipping the interceptor should not be the default behaviour, so we should add a new parameter:
```suggestion
  get(username: string, interceptorSkip = false) {
```